modules/platforms/dotnet/Apache.Ignite.Tests/Table/SchemaSynchronizationTest.cs

```diff
@@ -339,7 +339,6 @@ public async Task TestClientUsesLatestSchemaOnReadKv()
         [Values(true, false)] bool insertNewColumn,
         [Values(true, false)] bool withRemove)
     {
-        using var metricListener = new MetricsTests.Listener();
```

Comment:
What's this about?

Reply:
This was added to work around a transaction abort that could happen due to a concurrent schema change and an implicit transaction (generated by the streamer)